app-backend/api/src/main/scala/project/Routes.scala

```diff
@@ -613,6 +620,8 @@ trait ProjectRoutes
       entity(as[AnnotationFeatureCollectionCreate]) { fc =>
         val annotationsCreate = fc.features map { _.toAnnotationCreate }
         onSuccess(
+          // if projectLayerIdO is not provided as the last param
```

Comment:
I'd stick these comments inside the AnnotationDao function instead of where it's used